packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -26,10 +16,9 @@ export default function ShareModal({
   row,
   onRequestClose,
   ...props
-}: ShareModalProps): ReactElement {
+}: ShareModalProps & ModalProps): ReactElement {
```

Comment:
Can we instead change the `ShareModalProps` interface to also extend `ModalProps`?

Reply:
The `ShareModalProps` is used somewhere and that part doesn't need the `ModalProps`. Maybe I should change the name to something else as it implies Modal and must have Modal properties 🤔